netlogo-gui/src/main/ide/JumpToDeclaration.scala

```diff
@@ -0,0 +1,88 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.ide
+
+import org.nlogo.core.{Femto, Token, TokenType, TokenizerInterface}
+import org.nlogo.editor.EditorArea
+
+
+object JumpToDeclaration {
```

Comment:
This object needs its own test. I would like to see `jumpToDeclaration` and `findTokenContainingPosition` tested. I know we haven't worked a lot on testing this summer, so please IM me or comment here if you have questions about what sorts of things these tests need to cover.